test/bond-manager.test.ts

```diff
@@ -507,4 +523,5 @@ describe('Bond Manager contract', () => {
     let curator: BondManager
     let collateralTokens: ExtendedERC20
     let creator: BondFactory
+    const redemptionReason = 'test redemption reason string'
```

Comment:
should this be uppercase?

Reply:
Good question ...probably.

What I've done elsewhere is move the const out of the `describe` and have it at the top of the class, where the name is uppercased.

Reply:
e.g. https://github.com/windranger-io/windranger-treasury/blob/9c7c7dc3a5e5bc035443dab95c4c680f911d3ee5/test/bond-mediator.test.ts#L38